### PR TITLE
Link directly to the admin, fix #134

### LINKED ISSUE #134: Update links to always redirect creation/edition to the admin
As a logged in user I want to create and edit my resources via the admin interface in order to be consistent.

### PR DESCRIPTION
**Pull request by davidbgk**
This is a work in progress, the admin part is missing and there is probably dead code and missing features everywhere.

### CODE REVIEW

File: udata/core/dataset/views.py [hunk 1]
```diff
@@ -73,11 +57,7 @@ class DatasetListView(SearchView):
 
 
 navbar = nav.Bar('edit_dataset', [
-    nav.Item(_('Descrition'), 'datasets.edit'),
-    nav.Item(_('Additionnal informations'), 'datasets.edit_extras'),
-    nav.Item(_('Resources'), 'datasets.edit_resources'),
     nav.Item(_('Issues'), 'datasets.issues'),
```
Comment (noirbizarre) [inline udata/core/dataset/views.py]: Why keeping this one only ?

File: udata/core/dataset/views.py [hunk 2]
```diff
@@ -120,65 +100,6 @@ def get_context(self):
...
-
-@blueprint.route('/<dataset:dataset>/edit/resources/',
-                 endpoint='edit_resources')
-class DatasetResourcesEditView(ProtectedDatasetView, EditView):
-    form = DatasetForm
-    template_name = 'dataset/edit_resources.html'
-
-
 @blueprint.route('/<dataset:dataset>/issues/', endpoint='issues')
 class DatasetIssuesView(ProtectedDatasetView, DetailView):
```
Comment (noirbizarre) [inline udata/core/dataset/views.py]: It can be remove too

File: udata/core/organization/views.py [hunk 3]
```diff
@@ -46,11 +46,6 @@ def set_g_user_orgs():
 
 
 navbar = nav.Bar('edit_org', [
```
Comment (noirbizarre) [inline udata/core/organization/views.py]: I think it can be removed too

File: udata/core/organization/views.py [hunk 4]
```diff
@@ -201,96 +196,6 @@ def filter_activities(self, qs):
...
-
-
-@blueprint.route('/<org:org>/edit/teams/', endpoint='edit_teams')
-class OrganizationEditTeamsView(ProtectedOrgView, EditView):
-    form = OrganizationForm
-    template_name = 'organization/edit_teams.html'
-
-
 @blueprint.route('/<org:org>/issues/', endpoint='issues')
 class OrganizationIssuesView(ProtectedOrgView, DetailView):
```
Comment (noirbizarre) [inline udata/core/organization/views.py]: It can be remove too

**Comment by davidbgk** [comment]
Merged into: https://github.com/etalab/udata/pull/144

**Pull request closed without merging**